src/run_statistics.rs

```diff
@@ -28,117 +40,21 @@ impl fmt::Display for RunStatistics {
             .to_string()
         };
 
-        writeln!(f, "Runtime: {}", runtime_display)?;
+        writeln!(f, "Runtime: {}\n", runtime_display)?;
 
-        Ok(())
-    }
-}
-
-#[cfg(test)]
-mod tests {
-    use super::*;
-    use anyhow::anyhow;
-    use std::{
-        path::{Path, PathBuf},
-        time::Instant,
-    };
-    use wasmtime::*;
-    use wasmtime_wasi::WasiCtxBuilder;
-
-    #[test]
-    fn test_benchmark_runtime_allowed() {
-        let benchmark = run_function(
-            Path::new("tests/benchmarks/hello_world.wasm").to_path_buf(),
-            Path::new("tests/benchmarks/hello_world.json").to_path_buf(),
-        );
-
-        assert!(benchmark.runtime <= benchmark.threshold);
-    }
-
-    #[test]
-    fn test_benchmark_runtime_not_allowed() {
-        let benchmark = run_function(
-            Path::new("tests/benchmarks/sleeps.wasm").to_path_buf(),
-            Path::new("tests/benchmarks/sleeps.json").to_path_buf(),
```

Comment:
Do you also need to commit these files?

Reply:
What do you mean? Should I hardcode the bytecode in the tests?